Source/MedPodMod.cs

```diff
@@ -96,7 +96,7 @@ static class Toils_Bed_FailOnBedNoLongerUsable_CustomFailConditionForMedPods
             static MethodBase TargetMethod()
             {
                 predicateClass = typeof(Toils_Bed).GetNestedTypes(AccessTools.all)
-               .FirstOrDefault(t => t.FullName.Contains("c__DisplayClass3_0"));
+               .FirstOrDefault(t => t.FullName != null && t.FullName.Contains("c__DisplayClass3_0"));
```

Comment:
Is this nullcheck necessary?

Most modders that I've seen targeting predicate methods seem to be happy with just going straight to `Contains()` without the additional nullcheck.